src/common/socket.h

```diff
@@ -59,7 +59,7 @@ static inline int poll(struct pollfd *pfd, int nfds,
   return WSAPoll(pfd, nfds, timeout);
 }
 static inline int inet_pton(int family, const char* addr_str, void* addr_buf) {
```

Comment:
Maybe be better to remove? Because WIndows / Unix both have inet_pton as comment said before

Reply:
Good find!  I removed the "static inline int inet_pton" function from socket.h and got a successful build on Windows.

Should I close this PR and resubmit a new PR with it removed?